### PR TITLE
1.3.3 release notes

### PR DESCRIPTION
**Pull request by brian-avery**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** an issue which caused Prometheus to install improperly when using istioctl x manifest apply. ([Issue 16970](https://github.com/istio/istio/issues/16970))
+- **Fixed** a bug where locality load balancing can not read locality information from the node. ([Issue 17337](https://github.com/istio/istio/issues/17337))
+- **Fixed** a bug where long-lived connections were getting dropped by the Envoy proxy as the listeners were getting reconfigured without any user configuration changes. This was due to an underlying issue related to unstable configuration serialization between Envoy and Pilot. ([Issue 17383](https://github.com/istio/istio/issues/17383), [Issue 17139](https://github.com/istio/istio/issues/17139))
+- **Fixed** a crash in experimental analyze command. ([Issue 17449](https://github.com/istio/istio/issues/17449))
+- **Fixed** istioctl x manifest diff to diff text blocks in ConfigMaps. ([Issue 16828](https://github.com/istio/istio/issues/16828))
+
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]: We need one more entry for https://github.com/istio/istio/issues/17699.
I asked @jplevyak to add an entry (cf. https://github.com/istio/proxy/pull/2455#pullrequestreview-300335509).
Maybe something like:
```
- **Fixed** a segmentation fault crash in the Envoy proxy. ([Issue 17699](https://github.com/istio/istio/issues/17699))
```

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+release: 1.3.3
+---
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.2 and Istio 1.3.3.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** an issue which caused Prometheus to install improperly when using istioctl x manifest apply. ([Issue 16970](https://github.com/istio/istio/issues/16970))
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]:
```suggestion
- **Fixed** an issue which caused Prometheus to install improperly when using `istioctl x manifest apply`. ([Issue 16970](https://github.com/istio/istio/issues/16970))
```

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- **Fixed** a bug where locality load balancing can not read locality information from the node. ([Issue 17337](https://github.com/istio/istio/issues/17337))
+- **Fixed** a bug where long-lived connections were getting dropped by the Envoy proxy as the listeners were getting reconfigured without any user configuration changes. This was due to an underlying issue related to unstable configuration serialization between Envoy and Pilot. ([Issue 17383](https://github.com/istio/istio/issues/17383), [Issue 17139](https://github.com/istio/istio/issues/17139))
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]: I would simplify:
```suggestion
	- **Fixed** a bug where long-lived connections were getting dropped by the Envoy proxy as the listeners were getting reconfigured without any user configuration changes. ([Issue 17383](https://github.com/istio/istio/issues/17383), [Issue 17139](https://github.com/istio/istio/issues/17139))
```

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- **Fixed** a crash in experimental analyze command. ([Issue 17449](https://github.com/istio/istio/issues/17449))
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]: For consistency:
```suggestion
- **Fixed** a crash in `istioctl x analyze` command. ([Issue 17449](https://github.com/istio/istio/issues/17449))
```

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- **Fixed** istioctl x manifest diff to diff text blocks in ConfigMaps. ([Issue 16828](https://github.com/istio/istio/issues/16828))
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]:
```suggestion
- **Fixed** `istioctl x manifest diff` to diff text blocks in ConfigMaps. ([Issue 16828](https://github.com/istio/istio/issues/16828))
```

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+## Minor enhancements
+
+- **Added** a daily performance benchmark
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]: This is not user-visible, so I'd remove it.

**Comment by rlenglet** [comment]
Can you also update the `full_version` number in `data/args.yaml` like in https://github.com/istio/istio.io/pull/5070? I think it's better to do it all in a single PR.

**Comment by brian-avery** [comment]
Sure. This is the PR to master though, which is already updated to 1.4. Will add that in the cherry pick branch.

**Comment by rlenglet** [comment]
> Sure. This is the PR to master though, which is already updated to 1.4. Will add that in the cherry pick branch.

Sure, you're right.

File: content/en/news/2019/announcing-1.3.3/index.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.2 and Istio 1.3.3.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** an issue which caused Prometheus to install improperly when using `istioctl x manifest apply`. ([Issue 16970](https://github.com/istio/istio/issues/16970))
+- **Fixed** a bug where locality load balancing can not read locality information from the node. ([Issue 17337](https://github.com/istio/istio/issues/17337))
+	- **Fixed** a bug where long-lived connections were getting dropped by the Envoy proxy as the listeners were getting reconfigured without any user configuration changes. ([Issue 17383](https://github.com/istio/istio/issues/17383), [Issue 17139](https://github.com/istio/istio/issues/17139))
```
Comment (rlenglet) [inline content/en/news/2019/announcing-1.3.3/index.md]: The indentation is off here.

File: content/en/news/2019/announcing-1.3.3/index.md (hunk 6 above)
Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.3/index.md]: Removed. Thanks.

File: content/en/news/2019/announcing-1.3.3/index.md (hunk 7 above)
Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.3/index.md]: Fixed. Thanks.

**Comment by brian-avery** [comment]
Added do not merge label in this PR so we can merge it onrelease.

**Comment by geeknoid** [comment]
@brian-avery Note that this will need to be merged into release-1.3 in order to be published to istio.io. And note that as part o merging the PR to that branch, you also need to update the full_version field in data/args.yaml.

**Comment by brian-avery** [comment]
@geeknoid Understood. Thanks. Added the cherry-pick to 1.3 label already, so as soon as this merges, that PR will open. Master already has data/args updated for 1.4 so I'm going to update the cherry-pick branch with the change.

File: content/en/news/2019/announcing-1.3.3/index.md (hunk 1 above)
Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.3/index.md]: Added. Thanks.

**Comment by brian-avery** [comment]
@googlebot I fixed it

**Pull request merged**